assets/styl/_modules/buttons-icon-socials.json

```diff
@@ -0,0 +1,18 @@
+{
+    "socials" : {
+        "messenger" : "Messenger",
+        "facebook" : "Facebook",
+        "twitter" : "Twitter",
+        "linkedin" : "LinkedIn",
+        "googleplus" : "Google Plus",
+        "pinterest" : "Pinterest",
+        "whatsapp" : "What's App",
+        "sms" : "SMS",
+        "hootsuite" : "Hootsuite",
+        "pocket" : "Pocket",
+        "github" : "Github",
+        "snapchat" : "Snapchat",
+        "whatsapp" : "Whatsapp"
+
+    }
+}
```

Comment:
You have the same array in 2 different files.
It would be better to declare modifier classes for social default/icon buttons, just like for default buttons: https://github.com/20minutes/colette/blob/master/assets/styl/_modules/_buttons.styl#L13-L25